README.md

```diff
@@ -179,6 +179,26 @@ and surround it with the code fence for Mermaid:
 * In case of doubt, you will want to test your diagrams in the
   [Mermaid Live Editor](https://mermaid-js.github.io/mermaid-live-editor).
 
+### Read from .mmd files
+
+Sometimes mermaid diagrams can get quite large and having them inline with your main markdown doc can make it hard to manage. If you prefer to keep your diagrams in seperate `.mmd` files, you can put the file path to your `.mmd` file where you would like to insert the diagram.
```

Comment:
That is the motivation for the pull request.

Reply:
@fralau will do